charts/alfresco-search-service/charts/alfresco-insight-zeppelin/templates/config.yaml

```diff
@@ -1,17 +1,15 @@
----
-# Defines the config for the Alfresco Insight Zeppelin App
+{{- if not .Values.repository.existingConfigMap.name }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "alfresco-insight-zeppelin.fullName" . }}-configmap
+  name: {{ template "alfresco-insight-zeppelin.deployment.name" . }}
   labels:
-    heritage: {{ .Release.Service }}
-    release: {{ .Release.Name }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    component: zeppelin
+    {{- include "alfresco-insight-zeppelin.labels" . | nindent 4 }}
 data:
-  REPO_HOST: "{{ template "alfresco-repo-host" . }}"
-  REPO_PORT: "{{ template "alfresco-repo-port" . }}"
-  {{- range $key, $value := .Values.environment }}
-  {{ $key }}: {{ $value }}
+  {{- with .Values.repository }}
+  REPO_HOST: >-
+    {{ template "alfresco-common.url.host" .url }}
```

Comment:
So we have different config keys for values based configuration or configmap based config. I know this is what was initially wriiten in thez JIRA but I'm wondering if it's a good idea